docs/sources/setup/kubernetes.md

```diff
@@ -16,6 +16,9 @@ You can deploy Beyla in Kubernetes in two separate ways:
 - As a Sidecar Container (recommended)
 - As a DaemonSet
 
+In the future, we will release a [Beyla operator](https://github.com/grafana/beyla-operator)
+but at the moment it is work in progress and its API is not stable enough. 
+
```

Comment:
The docs team has a policy of not promising anything. I'll remove this and the good to merge.